posters-overrides.json

```diff
@@ -425,7 +425,11 @@
    "number": 145
   },
   {
-   "number": 147
+   "number": 147,
+   "title": "Functional MRI connectivity accurately distinguishes cases with psychotic disorders from healthy controls, based on cortical features associated with brain network development",
+   "presenter": "Dr Sarah E Morgan",
+   "institution": "Cambridge University, UK",
+   "pdf": "https://github.com/SarahMorgan/OHBM2020/blob/master/Poster147_OHBM2020.pdf"
```

Comment:
only what needs to be changed should be listed... but I guess this should be ok as well